ansible/roles/software/datagov-deploy-fgdc2iso/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']
+).get_hosts('all')
+
+
+def test_tomcat(host):
+    tomcat = host.service('tomcat')
+
+    assert tomcat.is_running
+
+
+def test_port(host):
+    assert host.socket("tcp://0.0.0.0:8080").is_listening
```

Comment:
tomcat should only listen to localhost, since nginx is handling the external connections.

Reply:
the role comes with 0.0.0.0 and not configurable. I was relying on aws security groups to block 0.0.0.0:8080. If really needed, we can upload a custom server.xml file bind to ip 127.0.0.1 instead of default 0.0.0.0.

Reply:
@adborden which approach to enforce 127.0.0.1 binding? Using lineinfile to change the server.xml in place, or providing a new server.xml to overwrite it altogether. Neither is my favorite.

Reply:
made a feature request to upstream, the new feature is implemented by the owner. now we can configure it to use 127.0.01.